content/en/network_monitoring/cloud_network_monitoring/guide/detecting_application_availability.md

```diff
@@ -0,0 +1,125 @@
+title: Detecting Application Availability using Network Insights
+description: Use CNM to detect application availability
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/apm-cnm-application-debugging/'
+      tag: 'Blog'
+      text: 'Debug application issues with APM and Cloud Network Monitoring'
+    - link: 'https://www.datadoghq.com/blog/cnm-best-practices/'
+      tag: 'Blog'
+      text: 'Best practices for getting started with Datadog Cloud Network Monitoring'
+    - link: 'https://www.datadoghq.com/blog/monitor-containers-with-cnm/'
+      tag: 'Blog'
+      text: 'How to monitor containerized and service-meshed network communication with Datadog CNM'     
+---
+
+## Overview
+
+Modern applications are built in a modular distributed manner with microservices and stateless design principles. Networking connects the distributed components of an application, ensuring a seamless user and application experience. By capturing, analyzing, and correlating network metrics such as latency, packet loss, and throughput across different applications and services, Datadog's Cloud Network Monitoring (CNM) provides actionable insights for application developers and SRE teams.
+
+## Service discovery and connectivity
+
+CNM is designed to track traffic between entities and determine which resources are communicating and their health status.
+
+To examine the a basic traffic flow between entities, use the following steps:
+
+1. On the [Network Analytics page][1], set your **View clients as** and **View servers as** dropdown filters to group by `service` tags to examine a service-to-service flow, and observe the basic traffic unit: a source IP communicating over a port to a destination IP on a port.
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_service_service.png" alt="CNM analytics page, grouping by service to service with Client and Server IP highlighted">}}
+
+   Each row aggregates 5 minutes worth of connections. While you might recognize some IPs as specific addresses or hosts depending on your network familiarity, this becomes challenging with larger, more complex networks. Therefore, the most meaningful aggregation level is rolling up the host or container associated with these IPs to tags in Datadog, such as `service`, `availability zone`, `pod`, and so on.
+
+2. For example, to view the network traffic for all of your `orders-sqlserver*` pods by host and availability zone, use the following filters to narrow down your search results:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_host_az.png" alt="CNM analytics page, grouping by host and availability zone for specific client pod name">}}
+
+   This initial step allows you to begin monitoring your most complex networks and start gaining insights between endpoints in your environment, including VMs, containers, services, cloud regions, datacenters, and more.
+
+### Service-to-Service dependency tracking
```

Comment:
```suggestion
### Service-to-service dependency tracking
```

Reply:
Same comment as above about nouns and verbs - I think this could indicate more clearly that it's a place to learn about how to perform this kind of tracking!